tests/utils_test.py

```
@@ -231,14 +239,19 @@ def load_and_clean_xml(location):
     for comparison. The file content is cleaned from variable parts such as
     dates, generated UUIDs and versions
     """
-    with io.open(location, encoding='utf-8') as l:
-        content = l.read()
-    data = xmltodict.parse(content, encoding='utf-8')
 
-    if 'creationInfo' in data['SpdxDocument']['Document']:
-        del(data['SpdxDocument']['Document']['creationInfo'])
+    parser = xmlparser.Parser(Builder(), StandardLogger())
+    with io.open(location, encoding='utf-8') as f:
+        document, _ = parser.parse(f)
```

Maybe we should raise an error in case parsing fails?